### ISSUE TITLE
Casting a web page

### ISSUE THREAD

**Issue by Tapanila** [Tapanila/SharpCaster#72]
There has been few requests to cast a webpage.
This would probably need to build custom receiver which you can control to open web page

**Comment by robchrstr** [comment]
Would be great to have the possibility to send a web page.

**Comment by Palloquin** [comment]
This could be achieved rather simply

Someone could develop a single custom receiver that holds nothing more than a 100% height, 100% width iFrame. A call could set the SRC URL of the iFrame.

This application could be reused by anyone. it raises some questions though about who will host this and privacy concerns around a third party hosting it.

The alternative would be to build and host your own custom receiver application, it's dead simple for this scenario.

**Comment by tapanila** [comment]
I think so too.

I started my receiver application here:
https://github.com/tapanila/SharpCaster-Receiver

I also published it with application id:
FE058D36

It is hosted at
https://sharpcaster.azurewebsites.net/

Currently having problems on transferring data to the website. I think the SDK does not support simple types (just string) and only supports json. So have to make the message format object based instead of string based.

Then need to add the logic to load a web page and send a scroll events. Also some kind of refresh option
